larq/optimizers_v2.py

```diff
@@ -97,27 +211,22 @@ def _resource_apply_dense(self, grad, var):
         var_t = lq.math.sign(-tf.sign(var * m_t - threshold) * var)
         return tf.compat.v1.assign(var, var_t, use_locking=self._use_locking).op
 
-    @staticmethod
-    def is_binary(var):
-        return "/kernel" in var.name and "quant_" in var.name
+    def _resource_apply_sparse(self, grad, var, indices):
+        raise NotImplementedError()
 
     def get_config(self):
-        fp_optimizer_config = self.fp_optimizer.get_config()
         config = {
+            "name": self._name,
```

Comment:
This should be included in [`super().get_config()`](https://github.com/tensorflow/tensorflow/blob/6eac303dd8534e65905e379d7f42f7b4a06bebfe/tensorflow/python/keras/optimizer_v2/optimizer_v2.py#L703) already.
```suggestion
```

Reply:
Nice, looks like `TestBopOptimizer.test_bop_serialization` still passes when I remove this so I'll delete it.